src/core_modules/capture-core/components/WidgetEnrollment/Date/Date.component.js

```diff
@@ -99,21 +107,24 @@ const DateComponentPlain = ({
     return editMode ? (
         <div data-test="widget-enrollment-date">
             <div className={classes.inputField}>
-                <CalendarInput
-                    calendar="gregory"
-                    dense
-                    className={classes.calendar}
+                <DateField
+                    width={200}
+                    value={selectedDate}
+                    onBlur={dateChangeHandler}
                     label={dateLabel}
-                    date={selectedDate}
+                    dense
                     locale={locale}
-                    onDateSelect={dateChangeHandler}
                 />
```

Comment:
The calendar for the enrollment date and incident date in the WidgetEnrollment component is in ISO format. When selecting a date, the date in the input field is also displayed in ISO format.

https://github.com/user-attachments/assets/a9ebb3f8-4fdb-4b3c-9057-38bf86b368fd